pom.xml

```diff
@@ -0,0 +1,122 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
We do not really need this Java service, or? We can simply use a dedicated server of the CoreNLP project (https://stanfordnlp.github.io/CoreNLP/corenlp-server.html#dedicated-server).

Reply:
As far as I remember from what @umairq mentioned, we needed it because CoreNLP only finds the coreference chains and doesn't replace them. 
Maybe we can also use their dedicated server and do the replacement in the python side.
But maybe @umairq can shed some light there.